conftest.py

```diff
@@ -97,5 +97,33 @@ def job_2(user_HR):
                          post_until='2022-12-23',
                          is_active=True,
                          marked_count=0,
-                         apply_link=DEFAULT_APPLY_LINK)
+                         apply_link=DEFAULT_APPLY_LINK+'2')
+    return job
+
+
+@pytest.fixture
+def job_SoftwareEngineer(user_HR):
```

Comment:
This function name does not follow [python function name convention ](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)
and its not consistent with the reset of the project.
Please change it and other similar function names.

Also, it seems that you use the new fixtures only for the search page tests, please move them to the same file.